pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.kokuwa.maven</groupId>
+        <artifactId>maven-parent</artifactId>
+        <version>0.6.13</version>
+        <relativePath />
+    </parent>
+
+    <groupId>org.technologybrewery.helmdependency</groupId>
+    <artifactId>helm-dependency-maven-plugin</artifactId>
+    <version>6.13.0-SNAPSHOT</version>
```

Comment:
A: I would expect this to either match the helm-maven-plugin version or just be 1.0.0-SNAPSHOT

Reply:
Fixed.